packages/components/src/progress-bar/styles.ts

```diff
@@ -70,7 +69,7 @@ export const Indicator = styled.div< {
 					width: `${ INDETERMINATE_TRACK_WIDTH }%`,
 			  } )
 			: css( {
-					width: `${ value }%`,
+					width: 'var(--indicator-width)',
```

Comment:
Do we need to prefix this var somehow so it doesn't accidentally bump into another one? Sometimes we use `--wp-admin`, sometimes we use `--wp-components`, and sometimes `--wp`, so perhaps it can be something like `--wp-progressbar-indicator-width` or something? @mirka thoughts?

Reply:
We discussed this during our pairing session and settled on this shorter one. I think it's very unlikely to collide with other css vars, and its internal to a component that doesn't have any slots either. The cool thing about CSS vars is that they are local to the element tree where they are declared, so collisions are almost never a concern unless the tree contain external, user-provided elements.